docs/content/concepts/dagster-pipes/aws-emr.mdx

```diff
@@ -0,0 +1,193 @@
+---
+title: "Integrating AWS EMR with Dagster Pipes | Dagster Docs"
+description: "Learn to integrate Dagster Pipes with AWS EMR to launch external code from Dagster assets."
+---
+
+# AWS EMR & Dagster Pipes
+
+This tutorial gives a short overview on how to use [Dagster Pipes](/concepts/dagster-pipes) with [AWS EMR](https://aws.amazon.com/emr/).
+
+The [dagster-aws](/\_apidocs/libraries/dagster-aws) integration library provides the <PyObject object="PipesEMRClient" module="dagster_aws.pipes" /> resource, which can be used to launch AWS EMR jobs from Dagster assets and ops. Dagster can receive regular events such as logs, asset checks, or asset materializations from jobs launched with this client. Using it requires minimal code changes to your EMR jobs.
+
+---
+
+## Prerequisites
+
+- **In the orchestration environment**, you'll need to:
+
+  - Install the following packages:
+
+    ```shell
+    pip install dagster dagster-webserver dagster-aws
+    ```
+
+    Refer to the [Dagster installation guide](/getting-started/install) for more info.
+
+  - **AWS authentication credentials configured.** If you don't have this set up already, refer to the [boto3 quickstart](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/quickstart.html).
+
+- **In AWS**:
+
+  - An existing AWS account
+  - Prepared infrastructure such as S3 buckets, IAM roles, and other resources required for your EMR job
+
+---
+
+## Step 1: Install the dagster-pipes module
```

Comment:
Can we be explicit that this is installing it in the EMR environment?
```suggestion
## Step 1: Install the dagster-pipes module in your EMR environment
```

Reply:
same, fixed in all tutorials